base/linalg/triangular.jl

```diff
@@ -1038,6 +1038,11 @@ for (f, g) in ((:/, :A_rdiv_B!), (:A_rdiv_Bc, :A_rdiv_Bc!), (:A_rdiv_Bt, :A_rdiv
         end
     end
 end
+### Fallbacks brought in from linalg/bidiag.jl while fixing #14506.
+# Eventually the above promotion methods should be generalized as
+# was done for bidiagonal matrices in #14506.
+At_ldiv_B(A::AbstractTriangular, B::AbstractVecOrMat) = At_ldiv_B!(A, copy(B))
+Ac_ldiv_B(A::AbstractTriangular, B::AbstractVecOrMat) = Ac_ldiv_B!(A, copy(B))
```

Comment:
I think I see the disconnect. Each generic promotion method in the section you linked contains a line like
```julia
TAB = typeof((zero(TA)*zero(TB) + zero(TA)*zero(TB))/one(TA))
```
or without the `one` in the non-quotient cases. So each such method requires definition of `zero`, and in the quotient cases also `one`, for `T`. But `T` does not necessarily have `one` and/or `zero` methods, for example where `A` is a matrix of matrices. Hence the need for more generic fallbacks like the two lines in question. Does that clarify? Thanks for bearing with me!